src/api/routes/users.js

```diff
@@ -85,6 +85,31 @@ router.post('/login', async (req, res, next) => {
   }
 });
 
+/**
```

Comment:
Note the comment in the OpenAPI definition

Reply:
fixed with PATCH /user/${id}